docs/index.js

```diff
@@ -19,6 +19,7 @@ import {
 } from '@deque/cauldron-react';
 import logo from './assets/img/logo.svg';
 import 'fontsource-roboto';
+import 'fontsource-lato';
```

Comment:
Interesting, tables are using a different font?

Reply:
yea, the `<TableHeader />` s are `Lato` according to the uxpin